nens_auth_client/views.py

```diff
@@ -120,7 +123,7 @@ def authorize(request):
     if user is None and request.session.get(INVITATION_KEY):
         try:
             invitation = Invitation.objects.select_related("user").get(
-                slug=request.session[INVITATION_KEY]
+                slug=request.session.pop(INVITATION_KEY)
```

Comment:
Dat het uit de sessie wordt gehaald: prima.
Het is wel een beetje een zij-effect dat 'ie verdwijnt. Nou gebruik je hem in de rest van de functie niet, maar ik kan me voorstellen dat het met debuggen op een gegeven moment raar is ("er is geen invitation key meer?!?").

OF later expliciet doen OF even een kort commentaarregeltje?